x-pack/plugin/stack/src/main/java/org/elasticsearch/xpack/stack/LegacyStackTemplateRegistry.java

```diff
@@ -179,7 +136,12 @@ protected List<LifecyclePolicy> getLifecyclePolicies() {
                 "/logs@settings.json",
                 REGISTRY_VERSION,
                 TEMPLATE_VERSION_VARIABLE,
-                ADDITIONAL_TEMPLATE_VARIABLES
+                Map.of(
```

Comment:
@felixbarny I am enabling usage of LogsDB also when `LegacyStackTemplateRegistry` is used...I was wondering...do we need to actually consider this option. Or can we just always inject "standard" for the legacy one?

Reply:
If I understand correctly the two registries exists just because each of them loads a set of things (templates, component templates and so on). AT the end the plugin uses both just to make sure everything is loaded, including legacy and new stuff so that users still using legacy naming conventions do not have issues.

Reply:
I think there are no integrations using the legacy component templates. I'd be fine with just using "standard" for the legacy templates. But making it dynamic is good, too.

Reply:
@salvatore-campagna given this statement, I think we can undo the changes to this file?

Reply:
I still need to inject "standard" using the map so that the json is well formed anyway. I can just remove the logic to evaluate the value based on setting.

Reply:
> I still need to inject "standard" using the map so that the json is well formed anyway. 

👍